tests/yam/agama/agama_first_boot.pm

```diff
@@ -0,0 +1,21 @@
+## Copyright 2023 SUSE LLC
```

Comment:
what doesn't make sense is that you take as the default scenario the one encrypted, should be the opposite.
You need a synchronization point in the single module [agama.pm](http://agama.pm/) that can process two type of needles encrypted-disk-password-prompt and grub2-agama.

And then driven by the schedule the next test module will handle that accordingly, if it is grub with grub action, if it is passphrase entering it. Therefore this file will not be needed.